libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ModelOverview.styles.ts

```diff
@@ -60,7 +60,10 @@ export const modelOverviewStyles: () => IProcessedStyleSet<IModelOverviewStyles>
         selectors: {
           "[role='tablist'].ms-Pivot": {
             display: "flex",
-            flexWrap: "wrap"
+            flexWrap: "wrap",
```

Comment:
I think you could refactor this into a common place and then use mergeStyles in here and other .styles.ts files to remove some duplicate lines, but maybe it's not worth the effort

Reply:
Thanks Ilya, great comment! I will check this in coming PR.